tslint.json

```diff
@@ -27,10 +27,10 @@
       "ignore-bound-class-methods"
     ],
     "switch-default": false,
-    "trailing-comma": [
-      true,
-      { "multiline": "always", "singleline": "never" }
-    ],
-    "no-unused-variable": [true, {"ignore-pattern": "React" }]
+    "no-unused-variable": [true, {"ignore-pattern": "React" }],
+    "trailing-comma": [false],
+    "no-console": [
+      ["error",  ["warn", "error"]]
+    ]
```

Comment:
Shouldn't affect us day to day, the old way would be that `tslint --fix` is done by the vscode extension would add the `,` for you, but now prettier is doing it.

The prettier part of CI would catch it if you shipped without the trailing comma now.